Automattic/jetpack
# blocks: remove BlockIcon wrapper in multiple blocks.

Follow-up from #13324

#### Changes proposed in this Pull Request:

All of these are passed as an icon prop to Placeholder components, which accepts a string, or ReactElement.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* check all those blocks in the editor, the icons should be displayed everywhere.
* Ensure that the icons are still styled properly, that no added margin / padding appear (see #12160)
